src/lidar/Manager.ts

```diff
@@ -277,7 +277,11 @@ export class LidarprofileManager {
       this.isPlotSetup_ = false;
     }
 
-    d3select('#gmf-lidarprofile-container .lidar-error').style('visibility', 'hidden');
+    const lidarContainerElement = document
+      .querySelector('gmf-desktop-canvas gmf-lidar-footer')
```

Comment:
```suggestion
      .querySelector('#gmf-lidar-footer')
```
I would use an id here. That would limit "not element found" errors.
Same below, do a git grep please